Sources/Sentry/Public/SentryOptions.h

```diff
@@ -191,6 +191,13 @@ NS_SWIFT_NAME(Options)
  * <code>YES</code>.
  */
 @property (nonatomic, assign) BOOL enableUIViewControllerTracking;
+
+/**
+ * Automatically attaches a screenshot when an error or exception is captured.
```

Comment:
`l`: Avoiding the passive voice is often good because it forces you to specify the subject. Especially for docs and requirements, this is nice because it makes them more specific and leaves less room for interpretation.
```suggestion
 * Automatically attaches a screenshot when capturing an error or exception.
```